templates/match_moved.tmpl

```diff
@@ -0,0 +1,9 @@
+{{define "content"}}
+<h3>Match has been moved</h3>
+<div>
+Due to space constraints, old matches are packaged and moved to <a href="http://data.lczero.org/files/">another server</a> every few days.
```

Comment:
I think its actually every few hours.

Reply:
Last time I checked it was about 2 days, but maybe saying `periodically` is more accurate.

Reply:
Sounds good.